### ISSUE TITLE
What's the final loss at the end of all epoches?

### ISSUE THREAD

**Issue by zidonghua2018** [ml-struct-bio/cryodrgn#459]
I trained cryodrgn and drgnai using dataset empiar10076, but the loss seemed not to decrease. The losses are 0.655 for cryodrgn and 1.022 for drgnai, but after a few epoches, the loss do not decrease much (0.635 for cryodrgn and 0.99 for drgnai after 3 epoches)

**Comment by michal-g** [comment]
Hi, changes in loss between training epochs do not necessarily correlate with updates to reconstruction quality nor model convergence due largely to the high-noise regime under which cryo-EM reconstruction models operate. We recommend instead comparing the outputs of the models themselves, as loss can indeed stop decreasing by significant amounts well before the model has stopped making significant improvements to these outputs!

Have you looked at the output volumes — and if doing heterogeneous reconstruction, the latent spaces — produced by tools such as `analyze` on various epochs from your models? This should give a much better idea of how your models are performing than only looking at the training loss! FYI, generally running cryoDRGN on the full 50S dataset takes at least 5-10 epochs to start converging.

Let us know if you have any other questions!
